### DISCUSSION TITLE
Gamepad controls Windows11

### DISCUSSION THREAD

**Discussion by Cioffaz** [nefarius/HidHide#133] [Q&A] [answered]
Hello, newcomer here!
Just a brief description of my setting. I've installed Yuzu emulator on my pc, in addition to Playnite (to organize games) and Sunshine (to stream the desktop and Playnite to the television). In the television i've installed Moonlight client (to receive the stream from Sunshine) and i've connected two bt controller (8bitDo). All is fully functional, without issue and the controllers are discovered and usable with the tv, with Playnite and the emulator.

The issue is: the controllers control also some part on Windows11, or other applications (games, Windows control panel, other uwp applications). Is there a way with HidHide to mask this behavior? When my daughter plays on tv i lose my pc!

Thank you!

**Comment by nefarius** [comment, marked as answer]
Sadly not, HidHide can only block traditional Win32 processes, it can not block other core system services that might detect and communicate with the controller over different channels.

**Reply by Cioffaz** [reply]
Thank you a lot for your work and the reply!

**Reply by joshkay** [reply]
Did you end up figuring out a solution to this?
